dev/com.ibm.ws.connectionpool.monitor/src/com/ibm/websphere/connectionpool/monitor/ConnectionPoolStatsMXBean.java

```diff
@@ -13,9 +13,44 @@
 import com.ibm.websphere.monitor.jmx.StatisticsMeter;
 
 /**
- * Management interface for the MBean "WebSphere:type=ConnectionPoolStats".
- * The Liberty profile makes this MBean available in its platform MBean server when the monitor-1.0 feature is
- * enabled to allow monitoring of the connection pool.
+ * <p>Management interface for the connection pool statistics MBean. One connection pool MBean instance exists per datasource or connection factory,
+ * whether configured explicitly in server configuration or whether it exists implicitly due to
+ * <code>@DataSourceDefinition</code> or <code>@ConnectionFactoryDefinition</code> annotation. The Liberty profile makes this MBean available in its platform MBean server when the
```

Comment:
This sentence sounds kind of awkward.  Maybe adding an "a" before `<code>@DataSourceDefinition`.... would help?